seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -1596,7 +1596,7 @@
         "Here's an example of a radio button:",
         "<code>&#60;label&#62;&#60;input type=\"radio\" name=\"indoor-outdoor\"&#62; Indoor&#60;/label&#62;</code>",
         "<hr>",
-        "Add a pair of radio buttons to your form. One should have the option of <code>indoor</code> and the other should have the option of <code>outdoor</code>."
+        "Add a pair of radio buttons to your form. One should have the option of <code>indoor</code> and the other should have the option of <code>outdoor</code>. All two should share the <code>name</code> attribute of <code>indoor-outdoor</code>"
```

Comment:
One more question @systimotic:

Do you think that `"Both buttons"` is clear enough to refer to `radio buttons`?

I'm raising that question because folks that hasn't English as their mother tongue can be confused. That is exact my case :)

I never thought in a radio button "as a button". Then, read `both buttons` seems to me that we are talking about another thing. If you just say to me `Both should ...` I understand quickly that both refer to radio buttons. But, for some reason the inclusion of buttons in the sentence makes me read again to be sure if I'm not missing anything. And as FCC has a huge community of no native English speakers, maybe others can experience the same difficulty on interpret the sentence.

What do you think?

BTW: Thanks for your comments about my profile. It's difficult to keep doing in that frequency, but is rewarding.